### DISCUSSION TITLE
Error: License is Invalid (Stata)

### DISCUSSION THREAD

**Discussion by annaschultze** [opensafely/documentation#386] [Q&A] [answered]
After updating OpenSAFELY and pulling the latest docker image, an action which previously ran locally is now failing with the error message 'license is invalid' (images below). The action passes test on Github (https://github.com/opensafely/covid-vaccine-neuro-research/runs/3487722550 ; note, the overall tests fail as expected because there are dependencies in other actions that @JemmaW will update later on). From previous conversations I gather this might be due to me having StataSE locally (rather than MP?). I'm not quite sure why this would just have broken now though. Does anyone have ideas for how this can be resolved? 

![image](https://user-images.githubusercontent.com/59482746/131721486-d249f47c-9cad-44c4-8b56-235ba21db5c8.png)

![image](https://user-images.githubusercontent.com/59482746/131721511-c6158257-a736-49f4-861b-e2861fe818a1.png)

**Comment by evansd** [comment, marked as answer]
Sorry, this was a bug introduced in v1.10.1 of the opensafely tool and is now fixed in v1.10.2 so running `opensafely upgrade` should make the problem go away.

**Reply by annaschultze** [reply]
Thanks, can confirm this worked!
